packages/yarnpkg-pnp/sources/types.ts

```diff
@@ -61,10 +61,6 @@ export type RuntimeState = {
 
 // This is what the generation functions take as parameter
 export type PnpSettings = {
-  // Some locations that are not allowed to make a require call, period
-  // (usually the realpath of virtual packages)
-  blacklistedLocations?: Iterable<PortablePath>,
```

Comment:
This is breaking since it changes the types of the public interface, so anyone using the API directly will need to change their code from `blacklistedLocations: ...` to `incompatibleLocations: ...`